cd/python/docker/Dockerfile

```diff
@@ -27,6 +27,7 @@ ARG PYTHON=python3
 ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.9
 RUN apt-get update && \
+    apt-get install -y wget ${PYTHON}-dev gcc && \
```

Comment:
You should be able to remove the "${PYTHON}-dev", since we're building from source.